frontend/src/components/atoms/LargeButton/LargeButton.js

```diff
@@ -28,44 +28,46 @@ const Container = styled.button`
 
 const Label = styled.span`
   padding: 12px 16px;
-
-  font-family: ${({ theme }) => theme.fontFamily};
-  font-weight: ${({ theme }) => theme.fontWeight.regular};
   font-size: ${({ fontSize }) => fontSize};
-
   letter-spacing: 0.014em;
-
   color: ${({ color }) => color};
-  transition: all 0.2s ease-in-out;
+  transition: all 0.3s ease-in-out;
 `;
 
-const LargeButton = ({
-  text,
-  background,
-  color,
-  fontSize,
-  fullWidth,
-  onClick,
-}) => (
-  <Container background={background} fullWidth={fullWidth} onClick={onClick}>
-    <Label color={color} fontSize={fontSize}>{text}</Label>
+const LargeButton = ({ background, color, fontSize, fullWidth, mt, ml, onClick, ...props }) => (
+  <Container
+    type={props.type}
+    color={color}
+    background={background}
+    fullWidth={fullWidth}
+    onClick={onClick}
+    marginTop={mt}
+    marginLeft={ml}
+  >
+    <Label color={color} fontSize={fontSize} {...props} />
```

Comment:
Trochę dziwny zabieg, że reszta propsów jest przekazana do `Label`, jakie ma to zastosowanie?

Reply:
Pozwala to na przekazanie tekstu, jaki ma być wyświetlany na przycisku. Dzięki temu możliwy jest zapis 
`<LargeButton>Zaloguj</LargeButton>`
W innej sytuacji trzeba to przekazać jako osobny props => children i to generalnie było by ok, ale z racji, że trzeba zrobić walidację z PropTypes, to z kolei czepia się w storybook, że nie ma przekazanego tego props-a.
![image](https://user-images.githubusercontent.com/18381342/151262793-3796c43f-8b21-4008-a4e6-acdf0a56a5e0.png)

Reply:
Można w ten sposób przekazywać `children`:
```js
const Template = ({children, ...args}) => (
  <ThemeProvider theme={theme}>
    <AdvancedButton {...args}>{children}</AdvancedButton>
  </ThemeProvider>
);
```

Reply:
Poprawiłem